### PR TITLE
Add queries to build.bat and test.bat.

### PR DESCRIPTION
**Pull request by isturdy**

### CODE REVIEW

File: opencensus/copts.bzl [hunk 1]
```diff
@@ -33,7 +33,7 @@ load(
 
 DEFAULT_COPTS = select({
     "//opencensus:llvm_compiler": LLVM_FLAGS,
-    "//opencensus:windows": MSVC_FLAGS,
+    "//opencensus:windows": MSVC_FLAGS + ["/wd4715"],
```
Comment (g-easy) [inline opencensus/copts.bzl]: Please add a comment about what 4715 is.

Also: not needed in TEST_COPTS?

File: tools/appveyor/build.bat [hunk 2]
```diff
@@ -12,11 +12,17 @@ REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
 REM limitations under the License.
 
-REM TODO: Get everything to build on Windows:
-REM bazel build //...
+REM We need this to append a variable within a loop.
+setlocal enabledelayedexpansion
+
+REM TODO: Is there an easier way to convert lines to a space-separated list?
+SET BUILDABLES=
+FOR /F usebackq %%T IN (`bazel query "kind(rule, //...)" ^| FINDSTR /C:"\:_" /V`) DO (
```
Comment (g-easy) [inline tools/appveyor/build.bat]: Nice.

File: tools/appveyor/build.bat [hunk 3]
```diff
@@ -12,11 +12,17 @@ REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+    SET BUILDABLES=!BUILDABLES! %%T
+)
 
 REM TODO: Remove --output_user_root after https://github.com/bazelbuild/bazel/issues/4149 is fixed.
-bazel --output_user_root=c:/t/ build //opencensus/trace
+bazel --output_user_root=c:/t/ build %BUILDABLES%
```
Comment (g-easy) [inline tools/appveyor/build.bat]: This PR is great overall, but I'd prefer to not have appveyor failing while we work on other stuff.

Please do:
```
bazel --output_user_root=c:/t/ build //opencensus/trace
￼REM bazel --output_user_root=c:/t/ build %BUILDABLES%
```

And we'll incrementally add targets as we get them building on Windows.

(I'm worried if every PR and commit has a red X after it, people will stop paying attention to CI entirely)

File: tools/appveyor/test.bat [hunk 4]
```diff
@@ -12,12 +12,17 @@ REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+REM TODO: Is there an easier way to convert lines to a space-separated list?
+SET TESTS=
+FOR /F usebackq %%T IN (`bazel query "kind(test, //...)  except attr('tags', 'manual|noci', //...)" ^| FINDSTR /C:"\:_" /V`) DO (
+    SET TESTS=!TESTS! %%T
+)
 
 REM TODO: Remove --output_user_root after https://github.com/bazelbuild/bazel/issues/4149 is fixed.
-REM bazel --output_user_root=c:/t/ test //opencensus/trace:all
-echo TODO: Make all tests pass on Windows.
+bazel --output_user_root=c:/t/ test --test_output=errors %TESTS%
```
Comment (g-easy) [inline tools/appveyor/test.bat]: Ditto.

File: opencensus/copts.bzl (hunk 1 above)
Comment (isturdy) [reply opencensus/copts.bzl]: Commented ("not all control paths return a value", treated as an error). I added it to TEST_COPTS for good measure--I do not think any of our code exercises that warning.

File: tools/appveyor/build.bat (hunk 3 above)
Comment (isturdy) [reply tools/appveyor/build.bat]: Added. (and //opencensus/stats, which builds once the switch fallthrough errors were suppressed.)

File: tools/appveyor/test.bat (hunk 4 above)
Comment (isturdy) [reply tools/appveyor/test.bat]: Done.

File: tools/appveyor/test.bat [hunk 5]
```diff
@@ -12,12 +12,18 @@ REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+
+REM TODO: Is there an easier way to convert lines to a space-separated list?
+SET TESTS=
+FOR /F usebackq %%T IN (`bazel query "kind(test, //...)  except attr('tags', 'manual|noci', //...)" ^| FINDSTR /C:"\:_" /V`) DO (
+    SET TESTS=!TESTS! %%T
+)
 
 REM TODO: Remove --output_user_root after https://github.com/bazelbuild/bazel/issues/4149 is fixed.
-REM bazel --output_user_root=c:/t/ test //opencensus/trace:all
-echo TODO: Make all tests pass on Windows.
```
Comment (g-easy) [inline tools/appveyor/test.bat]: Optional: leave the echo as a reminder. :)

File: tools/appveyor/build.bat (hunk 3 above)
Comment (g-easy) [reply tools/appveyor/build.bat]: Oh, awesome! Thanks for that.

**Comment by g-easy** [comment]
Please update the PR/commit description.

File: tools/appveyor/test.bat (hunk 5 above)
Comment (isturdy) [reply tools/appveyor/test.bat]: Done.

**Pull request merged**